README.md

```diff
@@ -289,8 +289,41 @@ All requests over non-SSL connections will be met with a 308 Permanent Redirect.
 HTTP Basic is required for all requests. The password will be ignored, and the
 username should be a unique per client uuid.
 
-The list of allowed clients is part of the configuration, and is passed as a
-comma-separated list to the `ROUTEMASTER_CLIENTS` environment variable.
+All allowed clients are stored in redis. These clients can be listed, added
+and deleted using the `ROUTEMASTER_ROOT_KEY` environment variable. All
+authentication is done with basic auth
+
```

Comment:
Could we change this summary to something more similar to what has been written in the PR description?

E.g.

> All allowed clients are stored in Redis. A "root" uuid must be specified in the `ROUTEMASTER_ROOT_KEY` environment variable. This key has the perms to add/delete/list the client uuids. Clients must use a uuid created by this user, at the `/api_keys` endpoint.

... or something like that.